<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:  #000000;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="text_clock">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="text">
     <string>&lt;b&gt;??&lt;/b&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="text_current_temp_f">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>128</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#feae3c;</string>
    </property>
    <property name="text">
     <string>&lt;b&gt;??&lt;/b&gt;&lt;sup&gt;&amp;deg;F&lt;/sup&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_hi_or_wc">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>115</y>
      <width>20</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="text">
     <string>HI:</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_hi_or_wc">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>115</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="text">
     <string>&lt;b&gt;???&lt;/b&gt;&amp;deg;F</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_dewpoint">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>115</y>
      <width>20</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="text">
     <string>DP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_dewpoint">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>115</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="text">
     <string>&lt;b&gt;???&lt;/b&gt;&amp;deg;F</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_humidity">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>115</y>
      <width>20</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="text">
     <string>HU:</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_humidity">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>115</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="text">
     <string>&lt;b&gt;???&lt;/b&gt;%</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_pressure">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>135</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="text">
     <string>&lt;b&gt;???&lt;/b&gt; in.</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="text_visibility">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>135</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="text">
     <string>&lt;b&gt;???&lt;/b&gt; miles</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pressure">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>135</y>
      <width>20</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="text">
     <string>PR:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_visibility">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>135</y>
      <width>20</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="text">
     <string>VI:</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_precip">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>135</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="text">
     <string>&lt;b&gt;???&lt;/b&gt; in.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_precip">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>135</y>
      <width>20</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="text">
     <string>PC:</string>
    </property>
   </widget>
   <widget class="QLabel" name="icon_current">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="hline_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>155</y>
      <width>320</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="hline_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>35</y>
      <width>320</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="hline_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>320</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="vline_1">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>155</y>
      <width>1</width>
      <height>84</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="vline_2">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>155</y>
      <width>1</width>
      <height>84</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="text_forecast1_date">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>106</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="text">
     <string>&lt;b&gt;???&lt;/b&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="text_forecast2_date">
    <property name="geometry">
     <rect>
      <x>107</x>
      <y>160</y>
      <width>106</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="text">
     <string>&lt;b&gt;???&lt;/b&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="text_forecast3_date">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>160</y>
      <width>106</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="text">
     <string>&lt;b&gt;???&lt;/b&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="text_forecast1_temp">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>225</y>
      <width>106</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="text">
     <string>&lt;b style=&quot;color:#ff0000;&quot;&gt;???&amp;deg;&lt;/b&gt; &lt;span style=&quot;color:#aaa;&quot;&gt;|&lt;/span&gt; &lt;b style=&quot;color:#0074a2;&quot;&gt;???&amp;deg;&lt;/b&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="text_forecast2_temp">
    <property name="geometry">
     <rect>
      <x>107</x>
      <y>225</y>
      <width>106</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="text">
     <string>&lt;b style=&quot;color:#ff0000;&quot;&gt;???&amp;deg;&lt;/b&gt; &lt;span style=&quot;color:#aaa;&quot;&gt;|&lt;/span&gt; &lt;b style=&quot;color:#0074a2;&quot;&gt;???&amp;deg;&lt;/b&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="text_forecast3_temp">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>225</y>
      <width>106</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="text">
     <string>&lt;b style=&quot;color:#ff0000;&quot;&gt;???&amp;deg;&lt;/b&gt; &lt;span style=&quot;color:#aaa;&quot;&gt;|&lt;/span&gt; &lt;b style=&quot;color:#0074a2;&quot;&gt;???&amp;deg;&lt;/b&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="icon_forecast1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="icon_forecast2">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>180</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="icon_forecast3">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>180</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="text_wind_dir">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>43</y>
      <width>20</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#999;</string>
    </property>
    <property name="text">
     <string>???</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="text_wind_speed">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>62</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#fff;</string>
    </property>
    <property name="text">
     <string>&lt;b&gt;???&lt;/b&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_wind_mph">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>88</y>
      <width>40</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#999;</string>
    </property>
    <property name="text">
     <string>MPH</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
